### PR TITLE
Swenis Magic.

### PR DESCRIPTION
**Pull request by marpme**

### CODE REVIEW

**Review by SwenVanZanten — APPROVED** [review]
💯

File: docker-compose.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-        ports:
-            - '21102:21102' #P2P port
-            - '20102:20102' #RPC port
-        image: 'marpme/verge-core'
+  verge-core:
+    build: .
+    ports:
+      - '21102:21102' #P2P port
+      - '20102:20102' #RPC port
+    image: 'hdsign/verge-core'
```
Comment (marpme) [inline docker-compose.yml]: let's create a verge docker repo?

File: Dockerfile [hunk 2]
```diff
@@ -20,12 +20,16 @@ RUN apt-get -y install libzmq3-dev
 
 # switch to the freshly cloned verge project & compile it
 WORKDIR /app/verge
+RUN git checkout master
```
Comment (marpme) [inline Dockerfile:23]: Can we use the latest known tag here? 

```bash
# Get new tags from remote
git fetch --tags

# Get latest tag name
latestTag=$(git describe --tags `git rev-list --tags --max-count=1`)

# Checkout latest tag
git checkout $latestTag
```

Comment (SwenVanZanten) [reply Dockerfile:23]: Then you never have the latest. And you would want to lock the tag for specific versions

**Pull request merged**